core/jazz_admin-api/components/config.js

```diff
@@ -0,0 +1,31 @@
+/**
+	Admin API Config File
+  @module: config.js
```

Comment:
You should now be using this version - https://github.com/tmobile/jazz/blob/v1.7.1/core/jazz_usermanagement/components/config.js